src/test/coins_tests.cpp

```diff
@@ -708,7 +711,7 @@ static void CheckSpendCoins(CAmount base_value, CAmount cache_value, CAmount exp
     GetCoinsMapEntry(test.cache.map(), result_value, result_flags);
     BOOST_CHECK_EQUAL(result_value, expected_value);
     BOOST_CHECK_EQUAL(result_flags, expected_flags);
-};
+}
```

Comment:
Why was this changed?

Reply:
Because class declarations don't need semicolons, but I'll revert if you find it distracting